site/content/en/docs/manual/advanced/consensus.md

```diff
@@ -0,0 +1,116 @@
+   2. Gaussian (similar to OKS) is calculated between these approximated lines with the same number of points
+4. Skeleton type:
+   1. Convert the skeleton annotation into points
+   2. Find the distance between these two sets of points. A set of points represents a skeleton.
+   3. Here, while calculating Object Keypoint Similarity (OKS) between the points, the scale parameter is also passed, accounting for the spread of the skeleton. This scale is obtained from the bounding box formed around the skeleton.
+
+
+## Step by Step Guide
+1. Create a new task ![new task](/images/new_task_page.jpeg)
+2. Set a non zero value to `Consensus Jobs Per Regular Job` parameter under the `Advanced configuration`
+   1. ![advanced config page](/images/adv_conf_page.jpeg)
+   2. ![task_page_consensus](/images/task_page_consensus.jpeg)
+3. Open this task, and assign workers to the consensus jobs
+   1. ![consensus task](/images/consensus_task.jpeg)
+   2. ![assigned workers](/images/assigned_workers.jpeg)
+4. This are the annotations made by `worker 1`
+   1. ![frame 1](/images/worker_1_1.jpeg)
+   2. ![frame 2](/images/worker_1_2.jpeg)
+   3. ![frame 3](/images/worker_1_3.jpeg)
+5. This are the annotations made by `worker 2`
+   1. ![frame 1](/images/worker_2_1.jpeg) In this frame the annotations are significantly off by our expectations.
+   2. ![frame 2](/images/worker_2_2.jpeg) In this frame worker 2 hasn't annotated one of the dogs.
+   3. ![frame 3](/images/worker_2_3.jpeg)
+6. Click on `Merge Consensus Jobs`, this will merge all the consensus jobs into their regular job.
+   1. ![consensus merge button](/images/consensus_merge_button.jpeg)
+   2. ![consensus merge confirmation](/images/consensus_merge_confirmation.jpeg)
+   3. ![loading consensus merge](/images/loading_consensus_merge.jpeg) The button will be disabled until the previous merge action isn't completed.
+7. Below are the merged results
+   1. ![frame 1](/images/merged_1.jpeg)
+   2. ![frame 2](/images/merged_2.jpeg) Through consensus it got the annotation from worker 1's annotation
+   3. ![frame 3](/images/merged_3.jpeg)
+8. On clicking on `View Consensus Analytics` ![view consensus analytics](/images/view_consensus_analytics.jpeg)
+   1. ![consensus analytics](/images/consensus_analytics_1.jpeg) Here the score corresponding to the job (merged job) represents, the mean overlap (IoU) among the annotations in it's consensus jobs and the merged job.
+   2. ![consensus analytics conflicts](/images/consensus_analytics_1_conf.jpeg) There was an annotation made by worker 1 but not by worker 2, that annotation didn't had any matching annotations so it raised `No matching annotation` conflict. Similarly, there were two annotations in the first frame which didn't matched as they didn't satisfied the `Min Overlap Threshold` condition. Thus, 3 `No matching annotation` conflicts are raised.
```

Comment:
I think the example is a little bit hard to gauge, unless the quorum is set to 2. Basically, there is no other consensus measurement can be between only 2 sources. I suggest either to increase the number or sources to 3 or add more examples.